bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/pwfs/PwfsRule.java

```diff
@@ -45,14 +45,14 @@ public IP2Problems check(ObservationElements elements)  {
                 try {
                     pwfs.checkBoundaries(primary, ctx).foreach(bs -> {
                         switch (bs) {
-                            case inside:
+                            case nearBoundary:
                                 problems.addWarning(PREFIX + "WARN", "The " + pwfs.getKey() + WARN, targetNode);
```

Comment:
So, `checkBoundaries` was returning `inside` to mean inside the inner vignetting limit.  It looks like you've renamed this `nearBoundary` but the warning message is still about vignetting.  I think you want the `vignetting` enum value here.